xml/schema/types/oblocks-2-9-6.xsd

```diff
@@ -43,9 +43,9 @@
         <xs:element name="oblock" minOccurs="0" maxOccurs="unbounded">
           <xs:complexType>
             <xs:sequence>
```

Comment:
As I read the scheme, `<path>` and `<portal>` can appear multiple times. Does `xs:all` permit that?